app/views/projects/members/_updated_at.html.erb

```diff
@@ -0,0 +1,4 @@
+<span class="text-sm">
```

Comment:
**suggestion (if-minor):** This is exactly the same as `projects/updated_at.html`. Could you make a component that take two parameters like `description` and `updated` and use it in both places (and anywhere else that would need it?

Reply:
Created a component for displaying the updated at time ago in d212883497cdc5da02bc5ca354c0cea05d7f9afd

Reply:
Perfect, thanks